### ISSUE TITLE
[Event] Operation Gnomeregan (Completely Missing)

### ISSUE THREAD

**Issue by gtao725** [azerothcore/azerothcore-wotlk#1688]
##### SMALL DESCRIPTION:

Just like the event Zalazane's Fall (Horde Only Event ID: 61), this is the Alliance version of the pre-cataclysm event. It seems this event doesn't even exist in the AC database. Nor does it exist in https://wowgaming.altervista.org/aowow/?events.

However, this event was definitely there in patch 3.3.5, and I found the event in Warmane's Wotlk DB: http://wotlk.cavernoftime.com/events

##### EXPECTED BLIZZLIKE BEHAVIOUR:

https://wow.gamepedia.com/Operation:_Gnomeregan
https://youtu.be/7saEJx3kEfA

##### CURRENT BEHAVIOUR:

Event doesn't exist at all. Quests are in the DB but I guess it isn't scripted since the entire event is missing.

##### STEPS TO REPRODUCE THE PROBLEM:

1.  No such event exists in the current AzerothCore DB.
2. 
3. 

##### EXTRA NOTES:

I compared https://wowgaming.altervista.org/aowow/?events and http://wotlk.cavernoftime.com/events and begin to wonder is AzerothCore using the most up to date DB for 3.3.5? Even the Event IDs between the 2 databases are different...

##### BRANCH(ES):

master

##### AC HASH/COMMIT:

9eb6e9a

##### OPERATING SYSTEM:

Win 7/10/Svr 2016

##### MODULES:

N/A

##### OTHER CUSTOMIZATIONS:

N/A

**Comment by poszer** [comment]
Missing script

**Comment by BarbzYHOOL** [comment]
> I compared https://wowgaming.altervista.org/aowow/?events and http://wotlk.cavernoftime.com/events and begin to wonder is AzerothCore using the most up to date DB for 3.3.5? Even the Event IDs between the 2 databases are different...

In fact, this aowow DB on wowgaming is using a very old TC database, not AC database. Yes it's stupidly misleading :'(

**Comment by gtao725** [comment]
> > I compared https://wowgaming.altervista.org/aowow/?events and http://wotlk.cavernoftime.com/events and begin to wonder is AzerothCore using the most up to date DB for 3.3.5? Even the Event IDs between the 2 databases are different...
> 
> In fact, this aowow DB on wowgaming is using a very old TC database, not AC database. Yes it's stupidly misleading :'(

Ah ok, so then which event id is it for the event Operation: Gnomeregan?

**Comment by poszer** [comment]
> > > I compared https://wowgaming.altervista.org/aowow/?events and http://wotlk.cavernoftime.com/events and begin to wonder is AzerothCore using the most up to date DB for 3.3.5? Even the Event IDs between the 2 databases are different...
> > 
> > 
> > In fact, this aowow DB on wowgaming is using a very old TC database, not AC database. Yes it's stupidly misleading :'(
> 
> Ah ok, so then which event id is it for the event Operation: Gnomeregan?

Are you on our discord?

**Comment by BarbzYHOOL** [comment]
Operation gnomeregan is for the transition between WOTLk to Cata if i'm right, and it's not coded on any emulator (I guess)

**Comment by gtao725** [comment]
Yes indeed it is. But how come Zalazane's Fall is coded although not working 100%? That is the Horde version of wotlk to cata transition event. Both were added in 3.3.5.

**Comment by Pondaveia** [comment]
This one seems a hard one to script without sniffs...

**Comment by Kittnz** [comment]
I'm planning on fixing this on TC, fix the current issues on Zalazane's Fall and implement the rest of the event on the echo isles, got a sniff for that and seems okay to script. Operation Gnomeregan isn't sniffed but there are a few scripts out there with a lot of information, maybe i can make it work as well, it's the same guy who scripted both  (blizz) so abilitys etc will work the same.

**Comment by Pondaveia** [comment]
The hero we needed but didn't deserve

**Comment by poszer** [comment]
Also, working on it : 
https://image.prntscr.com/image/Gsq8Wd1gTu6isH1ERyRJrg.png
https://image.prntscr.com/image/A7mjU4mrRAWreGj3Hz4HiQ.png
https://image.prntscr.com/image/dNARr_YxRl_Az6rEuOWVmA.png

**Comment by Kittnz** [comment]
> 
> 
> Also, working on it :
> https://image.prntscr.com/image/Gsq8Wd1gTu6isH1ERyRJrg.png
> https://image.prntscr.com/image/A7mjU4mrRAWreGj3Hz4HiQ.png
> https://image.prntscr.com/image/dNARr_YxRl_Az6rEuOWVmA.png

403 Forbidden

**Comment by poszer** [comment]
> > Also, working on it :
> > https://image.prntscr.com/image/Gsq8Wd1gTu6isH1ERyRJrg.png
> > https://image.prntscr.com/image/A7mjU4mrRAWreGj3Hz4HiQ.png
> > https://image.prntscr.com/image/dNARr_YxRl_Az6rEuOWVmA.png
> 
> 403 Forbidden, have you been working on something already?

Already shared images private :smiley:

**Comment by Pondaveia** [comment]
Looks good poszer 💃

**Comment by BarbzYHOOL** [comment]
so whats up on this?

**Comment by Kittnz** [comment]
Still needs to be scripted @BarbzYHOOL

**Comment by poszer** [comment]
Will be opened PR tonight. Almost done

**Comment by wlasser** [comment]
Is this resolved?

**Comment by FALL1N1** [comment]
@wlasser @gtao725 can you guys test https://github.com/azerothcore/azerothcore-wotlk/pull/3678 , please??

**Comment by IcedQuinn** [comment]
Should this be closed?

**Comment by heyitsbench** [comment]
> Should this be closed?

I don't see why, given that the event is still missing, not to mention the bounty tied to it.

**Comment by FALL1N1** [comment]
https://github.com/azerothcore/azerothcore-wotlk/pull/10983 if anyone wants to continue it feel free to fork it, everything is working as expected with:
- some missing texts
- missing visuals/gameobjects
- maybe more npcs? (can't see shit on the old videos)

**Comment by heyitsbench** [comment]
If anyone's interested, now's the time to sniff this event on Classic, same with Zalazane's Fall.

**Comment by Kittnz** [comment]
It's already sniffed :)

**Comment by heyitsbench** [comment]
> It's already sniffed :)

Can never be too sniffed!
